sound/soc/sof/ipc4-topology.c

```diff
@@ -1749,6 +1750,15 @@ static int sof_ipc4_prepare_process_module(struct snd_sof_widget *swidget,
 	}
 
 	memcpy(cfg, data, control_data->data->size);
+
+	/* Update pipeline params from output format */
```

Comment:
does this conflict with @libinyang 's PR?

Reply:
Yeah, we do the same thing to update pipeline params, we have synced, and I will merge some of his code in this PR, then we can close https://github.com/thesofproject/linux/pull/4097